### PR TITLE
V3 Yaml Examples

### PR DESCRIPTION
**Pull request by darrelmiller**

### CODE REVIEW

**Comment by MikeRalphson** [comment]
LGTM

>Tests: 4 passing, 0 failing, 0 warnings

File: examples/v3.0/api-with-examples.yaml [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+                - {
+                    "versions": [
+                        {
+                            "status": "CURRENT",
+                            "updated": "2011-01-21T11:33:21Z",
+                            "id": "v2.0",
+                            "links": [
+                                {
+                                    "href": "http://127.0.0.1:8774/v2/",
+                                    "rel": "self"
```
Comment (raymondfeng) [inline examples/v3.0/api-with-examples.yaml]: Should we add `rel` to https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/versions/3.0.md#linkObject?

**Comment by fehguy** [comment]
Please don't merge until we sort out the `href` purpose :D

**Comment by RobDolinMS** [comment]
@fehguy Do we still need to sort-out `href` purpose?  
* If yes, would you please link the Issue or PR?
* If no, would you want to give this a LGTM?

**Comment by webron** [comment]
`examples` need to be updated to the new construct.

**Comment by MikeRalphson** [comment]
Responses Object keys must be quoted:

> This field MUST be quoted for compatibility between JSON and YAML (i.e. "200")

**Comment by RobDolinMS** [comment]
@darrelmiller Do you want to update the PR to address @webron's and @MikeRalphson's feedback, or would you like them to submit PRs?

**Comment by RobDolinMS** [comment]
TDC: @darrelmiller has a few edits for this and will comment when ready.

**Comment by darrelmiller** [comment]
Issues have been addressed.  I'm going to be reckless and merge this without @OAI/tdc confirmation, because it doesn't affect the spec directly.

**Comment by darrelmiller** [comment]
And yet, I completely missed fixing the things I was supposed to be fixing.  PSA: Don't merge PRs without at least two cups of coffee first.

**Comment by webron** [comment]
Should this be reverted then?

**Comment by darrelmiller** [comment]
@webron Nah.  The changes are good, they are just not complete.  I've almost finished doing the rest of the changes in a new PR

**Pull request merged**